cluster/gce/gci/configure.sh

```diff
@@ -185,14 +184,14 @@ function is-preloaded {
 }
 
 function split-commas {
-  echo $1 | tr "," "\n"
+  echo "$1" | tr ',' '\n'
 }
 
 function remount-flexvolume-directory {
   local -r flexvolume_plugin_dir=$1
-  mkdir -p $flexvolume_plugin_dir
-  mount --bind $flexvolume_plugin_dir $flexvolume_plugin_dir
-  mount -o remount,exec $flexvolume_plugin_dir
+  mkdir -p "$flexvolume_plugin_dir"
+  mount --bind "$flexvolume_plugin_dir" "$flexvolume_plugin_dir"
+  mount -o remount,exec "$flexvolume_plugin_dir"
```

Comment:
Should the references here have curly braces like so:

```
"${flexvolume_plugin_dir}"
```

Reply:
Why? They serve no purpose here.

```
$ flexvolume_plugin_dir='foo bar'
$ echo $flexvolume_plugin_dir
foo bar
$
```

Reply:
Not a technical thing in this case, just for stylistic consistency. Purely a nit.

Reply:
As there is a severe mix of using {} and not all over the shell scripts already, I will keep it without here just as it currently is on master, and just add the quotes to handle the whitespace & c:o case properly.

Reply:
there's a massive style mismatch, I think in the past some of us tried to push for always using braces just because a lot of people editing these are not all that familiar with shell and it at least doesn't break anything if you include them superflously.

at this point i'm happy to let shellcheck more or less cover it, consistency is also not as important in cluster/ which theoretically we probably will remove someday totally :tm: (in reality CI depends on it, and getting rid of it is non trivial / not really staffed).